apps/web/src/components/layout/components/LocalStudioHeader/BackButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { Text } from '@novu/novui';
+import { IconOutlineArrowBack } from '@novu/novui/icons';
+import { hstack } from '@novu/novui/patterns';
+
+type BackButtonProps = { onClick: () => void };
+
+export function BackButton({ onClick }: BackButtonProps) {
+  return (
+    <button
+      className={hstack({
+        cursor: 'pointer',
+        gap: 'margins.icons.Icon20-txt',
+        px: '75',
+        py: '25',
+        borderRadius: '75',
+        textStyle: 'text.secondary !important',
+        _hover: { bg: 'badge.border', '& p, & svg': { color: 'typography.text.main !important' } },
+      })}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** I'm wondering, since we keep repeating this pattern of a lot of "inline" styling, ‏if we should maybe start a pattern of extracting this css classnames, just so we keep up readability

Reply:
Definitely a good call-out! I need to extra this one specifically as a variant on the button, but a lot of these things are coming up because they're not really specced out in the design system, which would help us standardize them